components/nft-sale-widget/hooks/useMintSection.ts

```diff
@@ -89,7 +89,7 @@ export const useMintSection = (isRandom: boolean) => {
     };
 
     void fetchWhitelist();
-  }, [activeSale, getAddresses, isAddressesLoading]);
```

Comment:
yeah `getAddresses` here might be technically unnecessary as a dependency but that's not the cause of rerendering

Reply:
resolved it as we discussed! :)